stand/kshim/bsd_kernel.c

```diff
@@ -25,6 +25,8 @@
 
 #include <bsd_global.h>
 
+#include <sys/tslog.h>
```

Comment:
I doubt that these will work with this file. kshim is a special environment and it's hard to test. I doubt this would even build in that environment.